network/docs/pages/development/network.rst

```diff
@@ -0,0 +1,102 @@
+.. _nuts-network-development:
+
+Nuts Network development
```

Comment:
The dev docs might already be covered by other docs already in the repo?

Reply:
Yes they are, will clean up docs.